### PR TITLE
Fixing bug where center of circle approaches extremely large values out of bounds

### PR DESCRIPTION
**Pull request by lvmaoa**
The center of some circles sometimes reaches extreme values due to some quadratics being very extreme (very wide or thin)
- Added filter on x and y to ignore those extreme values

### CODE REVIEW

File: TrajectoryPredictor.py [hunk 1]
```diff
@@ -194,7 +194,8 @@ def draw_points_to_frame(self, points: np.array, frame: np.array,
         # Draw each point
         for pt in pts:
             x, y = pt[0]
-            cv2.circle(frame, (int(x), int(y)), 2, (255, 0, 0), 2)
+            if x is not None and x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
```
Comment (dhrumilp15) [inline TrajectoryPredictor.py]: Can we remove `x is not None` ? Are there cases when cv2.projectPoints returns None as an element in the output points?

Comment (lvmaoa) [reply TrajectoryPredictor.py]: Yup!!!

File: TrajectoryPredictor.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def draw_points_to_frame(self, points: np.array, frame: np.array,
         # Draw each point
         for pt in pts:
             x, y = pt[0]
-            if x is not None and x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
+            if x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
```
Comment (dhrumilp15) [inline TrajectoryPredictor.py]: how about... `0 <= x <= frame.shape[1] and 0 <= y <= frame.shape[0]`? (Test this tho...)

File: TrajectoryPredictor.py (hunk 1 above)
Comment (dhrumilp15) [reply TrajectoryPredictor.py]: I luv u henrychan

**Comment by dhrumilp15** [comment]
Awesome work!

**Pull request merged**